styles/main.css

```diff
@@ -275,3 +301,79 @@ summary::before {
 details[open] > summary::before {
   transform: rotate(90deg);
 }
+
+.contact {
+  display: flex;
+  flex-direction: column;
+}
+
+#bg-contact {
+  width: 320px;
+  background: #fff url("../images/bg-contact.png") no-repeat;
+  background-size: cover;
+}
+
+.contact h2 {
+  color: #fff;
+  text-align: center;
+  font-size: 36px;
+  line-height: 40px;
+  margin-top: 0;
+  margin-bottom: 10px;
+}
+
+.contact h4 {
+  height: 24px;
+  font-size: 18px;
+  font-weight: 500;
+  margin-bottom: 0;
+}
+
+.contact .content {
+  width: 280px;
+  margin-top: 100px;
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+.contact p {
+  color: #ebebff;
+  text-align: center;
+  font-size: 18px;
+  line-height: 28px;
+  font-weight: 400;
+}
+
+form {
+  color: #ebebff;
+  margin-left: 20px;
+}
+
+form input,
+form textarea {
+  font-family: Arial, Helvetica, sans-serif;
+  border-style: none;
+  border-radius: 5px;
+  height: 30px;
+  width: 275px;
+}
+
```

Comment:
- [ ] kindly add padding at the left of your inputs and the textarea so that the text inside looks the same as the template one. correct your design below :point_down: :point_down:
![Screenshot from 2021-12-08 00-11-52](https://user-images.githubusercontent.com/65068771/145114144-3f62ae14-b69b-4cd4-b9c7-1662333faad3.png)

Reply:
Done, thanks :-)